src/sap/fhir/model/r4/lib/FHIRRequestor.js

```diff
@@ -420,10 +422,15 @@ sap.ui.define([
 			return "";
 		}
 
-		mParameters._format = "json";
+		// according to fhir all these kinds shall be intrepreted as json
+		if (!mParameters._format || !(mParameters._format in ["json", "application/json", "application/fhir+json"])) {
```

Comment:
I would recommend to add a variable in the file scope `var aSupportedFormats = ["json", "application/json", "application/fhir+json"]; 
.....
var FHIRRequestor = function(sServiceUrl, oModel, bCSRF, sPrefer , aDefaultQueryParams)`.

Reply:
than create a private method or a function in the file scope which tests this
`isFormatSupported(mParameters._format)`

Reply:
done